Specs/README.md

```diff
@@ -0,0 +1,2 @@
+# cocoapods-private-repo
+A private repo to develop Sonar
```

Comment:
It might be good to include a little documentation here about how to publish new versions of specs on this private repo. Are we copying the files into the `Specs` directory to "publish" them, or using `pod repo push`?